dbt_subprojects/dex/models/dex_info.sql

```diff
@@ -212,13 +212,14 @@ FROM (VALUES
     , ('reservoir_swap', 'Reservoir Swap', 'Direct', 'reservoir0x')
     , ('inkyswap', 'InkySwap', 'Direct', 'inkyswap')
     , ('squidswap', 'SquidSwap', 'Direct', 'squidswap_ink')
-    , ('swanswap', 'SwanSwap', 'Direct', 'swan_swap')
+    , ('swanswap', 'SwanSwap', 'Direct', 'swan_swap')1
```

Comment:
Please fix this line as well @PatelPrinci 
```suggestion
    , ('swanswap', 'SwanSwap', 'Direct', 'swan_swap')
```